dev/configs/.eslintrc.cjs

```diff
@@ -169,7 +170,8 @@ module.exports = defineConfig({
         {
             files: ["./dev/**"],
             rules: {
-                "no-restricted-imports": "off"
+                "no-restricted-imports": "off",
+                "import/no-extraneous-dependencies": "off"
```

Comment:
What problems was this causing? It would be bad if we were importing anything in `attest` that is not specified in its dependencies.